libs/application/templates/parental-leave/src/lib/messages.ts

```diff
@@ -437,6 +448,78 @@ export const m = defineMessages({
   },
 })
 
+export const employerForm = defineMessages({
+  formTitle: {
+    id: 'pl.application:form.title',
+    defaultMessage: 'Samþykki vinnuveitanda fyrir umsókn um foreldraorlof',
```

Comment:
I don't know if foreldraorlof is the correct word here, It's more common to use fæðingarorlof